dyn_em/module_big_step_utilities_em.F

```diff
@@ -6312,6 +6313,24 @@ SUBROUTINE sixth_order_diffusion( name, field, tendency, MUT, dt,  &
       i_end   = ite
       j_start = jts
       j_end   = MIN(jde-1,jte)
+      IF ( config_flags%open_xs ) THEN
+      i_start = MAX(its,ids+3)
```

Comment:
@weiwangncar 
Wei,
Looking at the rest of the routine, there is indenting of either 2 or 3 spaces. Go ahead and indent the assignments inside of your IF tests.